### PR TITLE
feat(theme): dropup

### PR DESCRIPTION
**Pull request by jare-talend**
**What is the problem this PR is trying to solve?**

**What is the chosen solution to this problem?**
add css to customize the behavior  when we expand to the up

**Please check if the PR fulfills these requirements**
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/components/stories/ActionDropdown.js [hunk 1]
```diff
@@ -1,8 +1,6 @@
 import React from 'react';
-import { // eslint-disable-line import/no-extraneous-dependencies
-	storiesOf,
-	action,
-} from '@storybook/react';
+// eslint-disable-next-line import/no-extraneous-dependencies
```
Comment (jare-talend) [inline packages/components/stories/ActionDropdown.js]: prettier manages bad the comment with // eslint-disable-line. The comment move to the next line and triggers an eslint error.
https://github.com/prettier/prettier/issues/1267#issuecomment-294165170

File: packages/theme/example/index.html [hunk 2]
```diff
@@ -515,6 +515,95 @@ <h1 id="buttons">Buttons</h1>
               </div>
             </div>
 
+            <div id="btn-dropup-icons" style="margin-bottom: 15px;">
```
Comment (frassinier) [inline packages/theme/example/index.html]: If you add some markup, please don't forget to add `#btn-dropup-icons` selector to our screenshots file [here](https://github.com/Talend/ui/blob/master/packages/theme/screenshots.config.json)

File: packages/theme/src/theme/_dropdowns.scss [hunk 3]
```diff
@@ -52,3 +52,25 @@
 		}
 	}
 }
+
+.dropup {
+	&.btn-group > .dropdown-menu {
+		transform: translateY(17px);
+	}
+
+	& .dropdown-toggle .caret {
```
Comment (frassinier) [inline packages/theme/src/theme/_dropdowns.scss]: `&` is useless

File: packages/theme/src/theme/_dropdowns.scss [hunk 4]
```diff
@@ -52,3 +52,25 @@
 		}
 	}
 }
+
+.dropup {
+	&.btn-group > .dropdown-menu {
+		transform: translateY(17px);
```
Comment (frassinier) [inline packages/theme/src/theme/_dropdowns.scss]: 17px? #RandomNumber :)

File: packages/theme/src/theme/_dropdowns.scss [hunk 5]
```diff
@@ -52,3 +52,25 @@
...
+	}
+
+	& .dropdown-toggle .caret {
+		margin-bottom: 0;
+		transform: rotate(-45deg);
+	}
+
+	&.open {
+		&.btn-group > .dropdown-menu {
+			transform: translateY(2px);
```
Comment (frassinier) [inline packages/theme/src/theme/_dropdowns.scss]: 2px? #RandomNumber

**Pull request merged**